### PR TITLE
Landing page

### LINKED ISSUE #1093: Landing page link back to alternate original system
## Summary

The warning is not appearing on landing page.

## Design Detail

This text box provides context into the new form and an option to return to the CAFC FRS or Call the CAFC, alternatives to this new NCFRS version. 

Screen Shot 2020-01-14 at 3 54 24 PM

https://www.figma.com/file/gzgsQIGQbO1wXZQzEHXhoe/Sprint-3?node-id=567%3A273&viewport=3216%2C1162%2C0.88847416639328

## Drawbacks

Potential for drop off (some users may be more comfortable with the interface they know in CAFC)

### LINKED ISSUE #1303: Add new first page before landing page
## Summary

First page before starting online report will redirect users to call or to other reporting options.

## Design Detail

https://www.figma.com/file/R054FzUOrpP13oTbMsQevj/Sprint-5?node-id=1984%3A0

Screen Shot 2020-02-06 at 11 47 27 AM

Screen Shot 2020-02-06 at 11 47 34 AM

Subsequent landing page will be leaner -- with only the information needed before reporting online:

Screen Shot 2020-02-06 at 11 49 33 AM

Screen Shot 2020-02-06 at 11 49 39 AM

### PR DESCRIPTION
**Pull request by naomiseminega**
Fixes #1093
Fixes #1303

# Description
Added new landing page (pre-service page) and new start page, as per new figma designs.

# Any new packages installed?
None

# Checklist:

- [x] I have looked at my code on GitHub and it all looks good (ex: no random commented out code or console.logs)
- [x] I have added and needed tests for my changes (in particular for new screens)
- [x] I have added a comment to any confusing code

# Required followup work

> Is there anything related to this that still needs to be done (ex: documentation changes).
- New footer needs to be added to all pages
- Test needs to be added for start page
- Discussion needs to be had on where the 'Call now' should land. It currently redirects to CAFC page.

### CODE REVIEW

File: f2/src/LandingPage.js [hunk 1]
```diff
@@ -1,24 +1,27 @@
 /* eslint-disable react/no-unescaped-entities */
 import React from 'react'
+import { useLingui } from '@lingui/react'
 import { Route } from 'react-router-dom'
 import PropTypes from 'prop-types'
 import { Trans } from '@lingui/macro'
 import { P } from './components/paragraph'
 import { Button } from './components/button'
-import { H1 } from './components/header'
+import { H1, H3 } from './components/header'
```
Comment (sastels) [inline f2/src/LandingPage.js]: The `H3`s should be `H2`s for a11y (ie to keep the H1 -> H2 -> H3 -> etc hierarchy). We can style them to be the same size as `H2`s.

**Comment by naomiseminega** [comment]
> * could you include a short render test for the `StartPage` in this PR?
> * I'm not sure why, but the two new pages don't have the padding (/margin?) at the bottom that the other pages on the app have. might have something to do with the `shouldWrapChildren` prop to `<Stack>`? Maybe fix this if it's easy, if not we can do it in another PR

Will do :)

File: f2/src/components/container/index.js [hunk 2]
```diff
@@ -17,17 +17,31 @@ export const InfoCard = props => (
...
 
-export const ConditionalForm = ({ ...props }) => (
+export const LandingBox = props => (
+  <Stack
+    direction="column"
+    bg="gray.300"
+    borderColor="gray.400"
+    boxShadow="lg"
+    color="black"
+    p={(4, null, 4)}
```
Comment (sastels) [inline f2/src/components/container/index.js]: from LGTM: should be square brackets `p={[4, null, 4]}`

File: f2/src/StartPage.js [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+                  <Trans id="startPage.requirement1" />
+                </Li>
+                <Li>
+                  <Trans id="startPage.requirement2" />
+                </Li>
+                <Li>
+                  <Trans id="startPage.requirement3" />
+                </Li>
+                <Li>
+                  <Trans id="startPage.requirement3" />
```
Comment (sastels) [inline f2/src/StartPage.js]: duplicated `startPage.requirement3`

**Pull request merged**